spec/ParseLiveQuery.spec.js

```diff
@@ -733,9 +731,7 @@ describe('ParseLiveQuery', function () {
     });
     object.set({ foo: 'bar' });
     await object.save();
-    setTimeout(async () => {
-      done();
-    }, 1000);
+    setTimeout(done, 1000);
```

Comment:
unclear why this is 1000msec, reducing will probably not cause the test to fail but might falsely have it succeed..